docs/backuprestore-with-oadp.md

```diff
@@ -1,5 +1,24 @@
 # Backup and restore with the Openshift API for Data Protection(OADP)
 
+- [Backup and restore with the Openshift API for Data Protection(OADP)](#backup-and-restore-with-the-openshift-api-for-data-protectionoadp)
+  - [Overview](#overview)
+  - [Pre-Requisites](#pre-requisites)
+  - [LCA apply wave annotation](#lca-apply-wave-annotation)
+  - [LCA apply label annotation](#lca-apply-label-annotation)
+  - [Install OADP and configure OADP on target cluster via ZTP GitOps](#install-oadp-and-configure-oadp-on-target-cluster-via-ztp-gitops)
+    - [Prepare OADP install CRs](#prepare-oadp-install-crs)
+    - [Prepare DataProtectionApplication(DPA) CR and S3 secret](#prepare-dataprotectionapplicationdpa-cr-and-s3-secret)
+    - [Rollout OADP changes via ClusterGroupUpgrade(CGU)](#rollout-oadp-changes-via-clustergroupupgradecgu)
+  - [Manually install OADP and configure OADP on target cluster](#manually-install-oadp-and-configure-oadp-on-target-cluster)
+  - [OADP Configmap generation](#oadp-configmap-generation)
+    - [Platform backup and restore CRs](#platform-backup-and-restore-crs)
+    - [Application backup and restore CRs](#application-backup-and-restore-crs)
+    - [Create OADP configmap with backup and restore CRs](#create-oadp-configmap-with-backup-and-restore-crs)
+  - [Reference OADP configmap in IBU CR](#reference-oadp-configmap-in-ibu-cr)
+  - [LCA Init Monitor Watchdog Timeout](#lca-init-monitor-watchdog-timeout)
```

Comment:
broken anchor, `lca-init-monitor-watchdog-timeout` doesn't seem available in the doc

Reply:
Good catch!